### DISCUSSION TITLE
bir, üç, beş, sekiz, onüç, ?

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#196] [Zeka Sorusu] [answered]
bir, üç, beş, sekiz, onüç, ?

Soru işaretli yere hangi sayı gelecektir.

**Comment by alicinar11** [comment, marked as answer]
Bir 3 harfli üç 2 harfli 3+2=5 
Beş 3 harfli 5+3=8
Sekiz 5 harfli 8+5=13
Onüç 4 harfli 13+4=17
Cevap: 17
